lib/views.js

```diff
@@ -438,7 +437,7 @@ exports.tasks_messages = {
             // used for fetching a specific message based on uuid
             emit(msg.uuid, val);
```

Comment:
yeah we are. 
 - Api end point for getting single message by id: `GET /api/v1/messages/:id`
 - Also `updateMessage` uses it:
    - api uses it internally in sms protocol to update state (`api/controllers/sms-gateway.js`, l.47). 
    -  externally `PUT /api/v1/messages/state/:id'`